requirements-ml.txt

```diff
@@ -1,5 +1,6 @@
 scikit-learn>=0.23.2
 keras>=2.4.3
+rapidfuzz>=2.6.1
```

Comment:
we might need to put some error catching if this is the case, e.g. user doesn't install [ml]

Reply:
true true!

Reply:
```
try:
    from rapidfuzz import fuzz, process
except ImportError:
    raise ImportError("Please ensure you run `pip install requirements-ml.txt`")
```

in `column_name_model.py`, this is an option but I'd rather just document setup and installation well. In the docs we should just be explicit that user must install `requirements-ml.txt` for the `column_name_model.py`

Reply:
yes, def be explicit about it in the docs

Reply:
added the `require_model` too on this